doc/build-helpers/fetchers.chapter.md

```diff
@@ -58,9 +116,7 @@ fetchurl {
 }
 ```
 
-**This will reuse the old contents**.
-Remember to invalidate the hash argument.
-You can do this by setting the `hash` attribute to an empty string.
+**This will produce the same output as before!**.
```

Comment:
```suggestion
**This will produce the same output as before!**.
To determine the new hash, set it to an empty value (it's a fake hash, hence the name of the method):
```